contracts/FeralfileArtworkV4_1.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "./FeralfileArtworkV4.sol";
+
+contract FeralfileExhibitionV4_1 is FeralfileExhibitionV4 {
+    struct AdvancedData {
```

Comment:
Should be `advance` (no need `ed` that could be wrong). Please check all related things as well.